c2corg_ui/static/js/map/map.js

```diff
@@ -114,9 +143,19 @@ app.MapController = function($scope, mapFeatureCollection) {
       new ol.layer.Tile({
         source: new ol.source.OSM()
       })
-    ]
+    ],
+    view: new ol.View({
+      center: ol.extent.getCenter(app.MapController.DEFAULT_EXTENT),
+      zoom: app.MapController.DEFAULT_ZOOM
+    })
   });
 
+  /**
+   * @type {ol.View}
+   * @private
+   */
+  this.view_ = this.map.getView();
+
   // editing mode
   if (this['edit']) {
```

Comment:
Argh, the changes we have made together passes in the compiler but generates errors in the console. For instance:
> angular.js:13236 Error: [$compile:nonassign] Expression 'undefined' in attribute 'appMapEdit' used with directive 'appMap' is non-assignable!
http://errors.angularjs.org/1.5.0/$compile/nonassign?p0=undefined&p1=appMapEdit&p2=appMap
    at angular.js:68
    at parentSet (angular.js:9550)
    at parentValueWatch (angular.js:9563)
    at regularInterceptedExpression (angular.js:15214)
    at Scope.$digest (angular.js:16655)
    at Scope.$apply (angular.js:16928)
    at bootstrapApply (angular.js:1694)
    at Object.invoke (angular.js:4604)
    at doBootstrap (angular.js:1692)
    at bootstrap (angular.js:1712)